entrypoint.sh

```diff
@@ -74,6 +74,10 @@ elif [ "$APPLICATION_TYPE" = "compliance-import-ssg" ]; then
   exec bundle exec rake ssg:import_rhel_supported --trace
 elif [ "$APPLICATION_TYPE" = "sleep" ]; then
   while true; do sleep 60; done
+elif [ "$APPLICATION_TYPE" = "compliance-unit-tests" ]; then
+  bundle exec rake db:test:prepare --trace
```

Comment:
Is this bit then not needed, I presume ? I see it's hardcoded into the unit-tests script now.